syncd/RedisClient.cpp

```diff
@@ -231,6 +231,32 @@ void RedisClient::setDummyAsicStateObject(
     m_dbAsic->hset(strKey, "NULL", "NULL");
 }
 
+void RedisClient::setDummyAsicStateObjects(
+        _In_ const sai_object_id_t* objectVids,
+        _In_ size_t count)
```

Comment:
count should be first, please change that in all files you touched

Reply:
done